### ISSUE TITLE
Don't open Output everytime on "Installing C# dependencies"

### ISSUE THREAD

**Issue by letmaik** [OmniSharp/omnisharp-vscode#5058]
## Environment data
VS Code version: 1.64.2
C# Extension version: 1.24.0

## OmniSharp log

```
Installing C# dependencies...
Platform: win32, x86_64

Finished
```

## Steps to reproduce
Start VS Code.

## Expected  behavior
Installation should happen in the background, at most with a status bar message.

## Actual behavior
The Output pane pops up, which is annoying and unnecessary.

**Comment by C-SELLERS** [comment]
Yeah, forcing VSCode to switch to "C#" output seems unnecessary. Anybody have a workaround for this?

**Comment by C-SELLERS** [comment]
Looks like they addressed this in #5041 
Will have to wait for full release 👍🏽

**Comment by nohwnd** [comment]
@C-SELLERS released today.
